tsconfig.json

```diff
@@ -7,13 +7,6 @@
     "noUnusedLocals": true,
     "strict": true
   },
-  "include": [
-    "**/*.ts"
-  ],
-  "exclude": [
-    "node_modules",
-    "**/*.test.ts",
-  ],
```

Comment:
Why is this change required? Wouldn't removing the exclusion potentially slow the build down a bit?

Reply:
Not really. Defaults are somewhat identical. And excluding test files is actually wrong... as we need them transpiled for Jest.

Reply:
With `ts-jest` the test files get transpiled when running the tests. I excluded them from the regular build to avoid putting transpiled tests into the package published on NPM.